src/platform_impl/linux/wayland/window/state.rs

```diff
@@ -748,8 +766,8 @@ impl WindowState {
             pointer.pointer().set_cursor(
                 serial,
                 Some(surface),
-                cursor.hotspot_x / scale,
-                cursor.hotspot_y / scale,
+                (cursor.hotspot_x as f64 / scale).round() as i32,
+                (cursor.hotspot_y as f64 / scale).round() as i32,
```

Comment:
I think we should generally apply the same rounding formula here as for `inner_size`